radar-auth/src/main/java/org/radarbase/auth/authentication/TokenValidator.kt

```diff
@@ -22,21 +22,21 @@ class TokenValidator
 @JvmOverloads
 constructor(
     /** Loaders for token verifiers to use in the token authenticator. */
-    verifierLoaders: List<TokenVerifierLoader>,
+    verifierLoaders: List<TokenVerifierLoader>?,
     /** Minimum fetch timeout before a token is attempted to be fetched again. */
-    fetchTimeout: Duration = Duration.ofMinutes(1),
+    fetchTimeout: Duration = Duration.ofMillis(1),
     /** Maximum time that the token verifier does not need to be fetched. */
     maxAge: Duration = Duration.ofDays(1),
 ) {
-    private val algorithmLoaders: List<TokenVerifierCache>
+    private val algorithmLoaders: List<TokenVerifierCache>?
```

Comment:
Can be non-nullable combined with:
algorithmLoaders = verifierLoaders?.map { loader ->
   CachedValue(config, supplier = loader::fetch)
} ?: emtpyList()